registry/zookeeper/registry.go

```diff
@@ -261,6 +261,25 @@ func (r *zkRegistry) Register(conf common.URL) error {
 		r.cltLock.Unlock()
 
 		logger.Debugf("(ZkProviderRegistry)Register(conf{%#v})", conf)
+
+	case common.ROUTER:
+
+		key := conf.String()
+		r.cltLock.Lock()
+		_, ok = r.services[key]
+		r.cltLock.Unlock()
+		if ok {
+			return perrors.Errorf("Path{%s} has been registered", conf.Path)
+		}
+		err = r.register(conf)
+		if err != nil {
+			return perrors.WithMessagef(err, "register(conf:%+v)", conf)
+		}
+		r.cltLock.Lock()
+		r.services[conf.Key()] = conf
+		r.cltLock.Unlock()
+		logger.Debugf("(ZkRouterRegistry)Register(conf{%#v})", conf)
+
 	}
```

Comment:
Where is the listener of the path dubbo/{service}/router?

Reply:
I have deleted this listener, it seems useless , i will confirm with Zhiyuan